cmd/snap-confine/snap-confine.c

```diff
@@ -509,6 +542,7 @@ int main(int argc, char **argv) {
     // a while. Note, we keep CAP_SYS_ADMIN in permitted as it will be needed
     // later.
     debug("dropping caps");
+    sc_debug_capabilities("before dropping effective caps");
     if (cap_set_proc(caps_no_effective) != 0) {
```

Comment:
for v1, could we update the `caps_no_effective` set in the earlier setup to also drop SETUID & SETGID from permitted set?

Reply:
is this done?

Reply:
No, this was suggested as another hardening yesterday.

Reply:
Done

Reply:
I had another look, cap_no_effective only has CAP_SYS_ADMIN and CAP_DAC_OVERRIDE, so this is ok.